src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_imad.cpp

```diff
@@ -155,7 +155,7 @@ FullyConnectedKernelIMAD::FullyConnectedTuningData FullyConnectedKernelIMAD::Get
     }
 
     // In most cases SIMD8 works faster than SIMD16
-    tuning_data.sub_group_size = 8;
+    tuning_data.sub_group_size = IsSIMDSizeSupported(params.engineInfo, 8) ? 8 : 16;
```

Comment:
could you add a unittest for this? (or do we already have one?)

Reply:
hmm I thought more about this but as we don't have any unittest result on the next-gen HW, I think it is OK as-is.